src/HttpCache/PurgerInterface.php

```diff
@@ -17,6 +17,8 @@
  * Purges resources from the cache.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
+ *
+ * @deprecated use {@see TagsHeadersProviderInterface} and {@see TagsInvalidatorInterface} instead
```

Comment:
let's avoid deprecating this new interface, it was discussed pre-3.0 and I don't think the change is justified.